src/QMCHamiltonians/ForceCeperley.cpp

```diff
@@ -62,8 +81,11 @@ void ForceCeperley::InitMatrix()
 
 ForceCeperley::Return_t ForceCeperley::evaluate(ParticleSet& P)
 {
-  forces                               = forces_IonIon;
-  forces_ShortRange                    = forces_IonIon;
+  if (addionion == true)
+    forces = forces_IonIon;
+  else
+     forces = 0.0;
+  //forces                               = forces_IonIon;
```

Comment:
Please remove commented line. One line above the indent is not correct. If you have clang-format around, could you run it on this file?